src/eip7594/eip7594.c

```diff
@@ -454,6 +454,125 @@ static C_KZG_RET compute_r_powers_for_verify_cell_kzg_proof_batch(
     return ret;
 }
 
+/**
+ * Aggregate columns, compute the sum of interpolation polynomials, and commit to the result.
+ *
+ * This function computes: [sum_k r^k interpolation_poly_k(s)]
+ *
+ * This function first aggregates cells from the same column by scaling them with the corresponding
```

Comment:
I feel like this comment is not describing *what* the function does, but instead it describes *how* it does it.
Maybe as the user of the function I don't need to know that. So maybe move this comment into the function?